RebuildWithLocalMSBuild.cmd

```diff
@@ -1,26 +1,25 @@
 :: This script will:
-::   1) Rebuild MSBuild source tree.
-::   2) Create a copy of the build output in bin\MSBuild
+::   1) Build MSBuild source tree.
```

Comment:
Rainer told me to keep this as rebuild when I also renamed it to build on the xplat branch, since it ends up calling the Rebuild target :)